packages/aila/src/core/llm/MockLLMService.ts

```diff
@@ -44,8 +44,8 @@ export class MockLLMService implements LLMService {
       async start(controller) {
         for (const chunk of responseChunks) {
           controller.enqueue(chunk);
-          await sleep(0);
```

Comment:
This seem to resolve the fixture not coming back immediately. I can't remember the exact issue but it was something to do with await returning control to the outer closure. Removing this line stops that happening and the fixture comes down immediately.

Reply:
Ok. Maybe we can put a pin in it for now for an easy life, and see if we can make it more realistic later. It'll be nice to have this snappy

It would be useful to dive into problems like that in different PRs, but I realise that's tricky when rejigging the chat